tests/codegen/call_join.cpp

```diff
@@ -0,0 +1,27 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_join)
+{
+  test("struct arg { char **argv } kprobe:f { @x = (struct arg *) 0; "
+       "join(@x->argv); }",
```

Comment:
```suggestion
  test("struct arg { char **argv } kprobe:f { $x = (struct arg *) 0; "
       "join($x->argv); }",
```
Nit: To keep the unit tests as specific as possible - this will mean the only maps in the LLVM IR are the scratch maps